corehq/apps/users/views/__init__.py

```diff
@@ -267,6 +268,7 @@ def main_context(self):
     def page_context(self):
         ctx = {
             'form_uneditable': BaseUserInfoForm(),
+            'ews_extension': EWS_WEB_USER_EXTENSION.enabled(self.domain)
```

Comment:
You can avoid passing additional context to the template by using the `toggle_enabled` template filter:
```jinja
{% if request|toggle_enabled:"EWS_WEB_USER_EXTENSION" %}
```